orchestrators/ComplianceOperations/src/DataProviders/AmlDataProvider.php

```
@@ -0,0 +1,312 @@
+            $transactions,
+            $periodStart,
+            $periodEnd
+        );
+    }
+
+    /**
+     * Check if party is high risk.
+     *
+     * @param string $partyId Party identifier
+     */
+    public function isHighRisk(string $partyId): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$partyId'. (undefined)

(UnusedFormalParameter)